### DISCUSSION TITLE
Card Body Slam

### DISCUSSION THREAD

**Discussion by luix10** [oskarrough/slaytheweb#236] [General]
Hello, cool project, congrats and thanks for sharing it!
The card Body Slam wasn't working and I managed to make it work by adding the property damage:0 in the card file, don't know if its a known bug but some other cards also don't have the damage property and works fine...

**Comment by oskarrough** [comment]
Hey, thank you very much! Happy you like it.

The [Body Slam card](https://github.com/oskarrough/slaytheweb/blob/main/src/content/cards/body-slam.js) triggers this extra action called `dealDamageEqualToBlock`, but you're totally right. It doesn't deal damage as it should.

https://github.com/oskarrough/slaytheweb/blob/0b4b617db5ea3c20fc97fd516cee2ed555de2235/src/game/actions.js#L226

So it should work without adding `damage: 0`. When it does not, it sounds like you found a sneaky bug.  We could either close our eyes add `damage: 0` to the card and move on. Or figure out why that makes a difference.. it shouldn't be needed imo.

**Comment by oskarrough** [comment]
It was fixed here https://github.com/oskarrough/slaytheweb/pull/246
